src/main/io/dji_osd_symbols.h

```diff
@@ -62,20 +62,41 @@
 #define DJI_SYM_HEADING_DIVIDED_LINE    0x1C
 #define DJI_SYM_HEADING_LINE            0x1D
 
-// AH Center screen Graphics
+// AH Center Screen Graphics
 #define DJI_SYM_CROSSHAIR_LEFT          0x72
 #define DJI_SYM_CROSSHAIR_CENTRE        0x73
 #define DJI_SYM_CROSSHAIR_RIGHT         0x74
-#define DJI_SYM_AH_RIGHT                0x02
-#define DJI_SYM_AH_LEFT                 0x03
+#define DJI_SYM_AH_RIGHT                0x03
+#define DJI_SYM_AH_LEFT                 0x02
```

Comment:
Thanks 👍🏻

Reply:
Could you please let me know what your thoughs are on the `_CROSSHAIR_` and `_AH_CH_` as well?

Do you think that they should be chnaged (lines 66 to 68) in `dji_osd_symbols.h` and the `indisplayport_msp_dji_compat.c` respectively?